### PR TITLE
Nova db settings

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by vuntz** [comment]
+1

File: chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+def upgrade(ta, td, a, d)
+  unless a["db"].key? "max_pool_size"
+    a["db"]["max_pool_size"] = ta["db"]["max_pool_size"]
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb]: I wonder if we need this code when its nil in the json...

**Comment by dirkmueller** [comment]
+0.1. I'm a bit unsure about the migration (see inline comment) but the rest looks good.

**Comment by toabctl** [comment]
@vuntz still +1 ?

File: chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb (hunk 1 above)
Comment (toabctl) [inline chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb]: It worked, but I removed the whole upgrade code because.

**Comment by vuntz** [comment]
yes, still +1

**Comment by dirkmueller** [comment]
+1

File: chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+def upgrade(ta, td, a, d)
+  # NOTE(toabctl): we do nothing here because all the added keys
```
Comment (tboerger) [inline chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb]: I think we don't need this kind of comment...

Comment (aplanas) [inline chef/data_bags/crowbar/migrate/nova/037_add_sqlalchemy_pool_values.rb]: I find this useful!

**Pull request merged**